docs/manual/en/introduction/Installation.html

```diff
@@ -10,149 +10,245 @@
-		</p>
-
-		<p>
-			All methods of installing three.js depend on ES modules (see [link:https://eloquentjavascript.net/10_modules.html#h_hF2FmOVxw7 Eloquent JavaScript: ECMAScript Modules]), which allow you to include only the parts of the library needed in the final project.
-		</p>
-
-		<h2>Install from npm</h2>
+		<h2>Quickstart</h2>
 
 		<p>
-			To install the [link:https://www.npmjs.com/package/three three] npm module, open a terminal window in your project folder and run:
+			If you already know how to set up a project and install npm dependencies, you can skip the rest of this <i>Installation</i>  guide and install three.js with the line below. Otherwise, continue to <i>Project structure</i> section.
 		</p>
 
 		<code>
-		npm install three
+			npm install --save three
 		</code>
 
+		<h2>Project structure</h2>
+
 		<p>
-			The package will be downloaded and installed. Then you're ready to import it in your code:
+			Every three.js project needs at least one HTML file to define the webpage, and a JavaScript file to run your three.js code. The structure and naming choices below aren't required, but will be used throughout this guide for consistency.
 		</p>
 
-		<code>
-		// Option 1: Import the entire three.js core library.
-		import * as THREE from 'three';
+		<ul>
+			<li>
+				<i>index.html</i>
+				<code>
+		&lt;!DOCTYPE html&gt;
+		&lt;html lang="en"&gt;
+			&lt;head&gt;
+				&lt;meta charset="utf-8"&gt;
+				&lt;title&gt;My first three.js app&lt;/title&gt;
+				&lt;style&gt;
+					body { margin: 0; }
+				&lt;/style&gt;
+			&lt;/head&gt;
+			&lt;body&gt;
+				&lt;script type="module" src="/main.js"&gt;&lt;/script&gt;
+			&lt;/body&gt;
+		&lt;/html&gt;
+				</code>
+			</li>
+			<li>
+				<i>main.js</i>
+				<code>
+import * as THREE from 'three';
 
-		const scene = new THREE.Scene();
+...
+				</code>
+			</li>
+			<li>
+				<i>public/</i>
+				<ul>
+					<li>
+						The <i>public/</i> folder is sometimes also called a "static" folder, because the files it contains are pushed to the website unchanged. Usually textures, audio, and 3D models will go here.
+					</li>
+				</ul>
+			</li>
+		</ul>
+
+		<p>
+			Now that we've set up the basic project structure, we need a way to run the project locally and access it through a web browser.
+		</p>
+
+		<h2>Option 1: Installing with a build tool</h2>
+
+		<h3>Development</h3>
+
+		<p>
+			Installing from npm and using a build tool is the recommended approach for most users — the more dependencies your project needs, the more likely you are to run into problems that the static hosting cannot easily resolve. With a build tool, importing local JavaScript files and npm packages should work out of the box, without import maps.
+		</p>
+
+
+		<ol>
+			<li>
+				Install [link:https://nodejs.org/ Node.js]. We'll need it to load manage dependencies and to run our build tool.
```

Comment:
Maybe add a note here about package.json here? For new users..
Something like "You can a local project by running `npm init`. A `package.json` file will be created."
Or link to a guide explaining this?

Reply:
I don't think there's any reason users would need to run `npm init`, it's not required, and many of its inputs are for libraries not applications. But mentioning `package.json` will be created, and linking to an explanation of what that is, we could do. Does that sound OK?

Reply:
Yup, just a mention + link to guide would be enough!